src/ocean/util/app/ext/TimerExt.d

```diff
@@ -87,7 +87,7 @@ unittest
         }
     }
 
-    new App;
+    (new App).run(null);
```

Comment:
Not an unittest please, an integration test. `unittest` shouldn't do IO or affect global state, and `epoll` does.

Reply:
Well, there's no IO with this particular unittest. The `epoll_create` and `timerfd_create` calls already affect the global state, with one epoll cycle or not.

Reply:
Well, I think they shouldn't be called then :)

Reply:
Ok, I'll move the test into an integration test and remove `new App` from here.